code/modules/mob/mob_defines.dm

```diff
@@ -100,6 +100,7 @@
 	var/list/speak_emote = list("says") // Verbs used when speaking. Defaults to 'say' if speak_emote is null.
 	var/emote_type = 1		// Define emote default type, 1 for seen emotes, 2 for heard emotes
 	var/facing_dir = null   // Used for the ancient art of moonwalking.
+	var/canshoot = 1 // If they can shoot a gun
```

Comment:
Not a fan of this. Isn't there another solution, like forcing people to be empty-handed inside pods, and giving the pods storage space for guns?

Interesting that `resting = 1` doesn't stop people from firing weapons.